### DISCUSSION TITLE
best way to link to pretty html

### DISCUSSION THREAD

**Discussion by jdduprey** [sr320/course-fish546-2021#92] [Q&A] [answered]
Now that I have my multiqc html report what is the best way to link to the pretty version from github? I remember Steven briefly showed us a way to do this but I don't remember the specifics.

**Comment by yaaminiv** [comment, marked as answer]
2 options!

https://nbviewer.jupyter.org
https://htmlpreview.github.io

**Comment by jdduprey** [comment]
Thanks for the lightning fast response! The summary statistics appear, but some of the other figures seem to be stuck on loading? Any idea of what could be going on? Here's my [script](https://github.com/fish546-2021/joe-benthos/blob/main/code/04-gannet-to-multiqc.ipynb), and the [output](https://htmlpreview.github.io/?https://github.com/fish546-2021/joe-benthos/blob/main/analysis/fastqc-out/multiqc_report.html).

**Reply by yaaminiv** [reply]
Not sure what's happening either, but it loads in nbviewer: https://nbviewer.jupyter.org/github/fish546-2021/joe-benthos/blob/main/analysis/fastqc-out/multiqc_report.html

Sometimes things are just slow; there's nothing wrong that I can see with your output!

**Reply by jdduprey** [reply]
Oh, yeah it works in nb for me too, weird. Thanks Yaamini.
